cypress/e2e/Simple.cy.ts

```diff
@@ -1,14 +1,130 @@
+    cy.get('[data-cy="Column Annotation-step"]').within(() => {
+      cy.get('.MuiStepLabel-iconContainer').should('have.class', 'Mui-active');
+    });
+    cy.get('[data-cy="1-column-annotation-card"]').should('be.visible');
+    cy.get('[data-cy="2-column-annotation-card"]').should('be.visible');
+    cy.get('[data-cy="3-column-annotation-card"]').should('be.visible');
+    cy.get('[data-cy="column-annotation-pagination"]').should('be.visible');
+    cy.get('[data-cy="2-description"]').should('contain', 'Age of the participant');
+    cy.get('[data-cy="2-edit-description-button"]').click();
+    cy.get('[data-cy="2-description-input"]')
+      .should('be.visible')
+      .and('contain', 'Age of the participant');
+    cy.get('[data-cy="2-description-input"]').clear();
+    cy.get('[data-cy="2-description-input"]').type('some cool new description');
+    cy.get('[data-cy="2-save-description-button"]').click();
+    cy.get('[data-cy="2-description"]').should('contain', 'some cool new description');
+    cy.get('[data-cy="1-column-annotation-card-data-type-continuous-button"]').click();
+    cy.get('[data-cy="2-column-annotation-card-data-type-continuous-button"]').click();
+    cy.get('[data-cy="3-column-annotation-card-data-type-categorical-button"]').click();
+    cy.get('[data-cy="1-column-annotation-card-standardized-variable-dropdown"]').type(
+      'participant{downArrow}{enter}'
+    );
+    cy.get('[data-cy="2-column-annotation-card-standardized-variable-dropdown"]').type(
+      'age{downArrow}{enter}'
+    );
+    cy.get('[data-cy="3-column-annotation-card-standardized-variable-dropdown"]').type(
+      'sex{downArrow}{enter}'
+    );
     cy.get('[data-cy="next-button"]').click();
+
+    // Value Annotation view
     cy.contains('Value Annotation');
+    cy.get('[data-cy="Value Annotation-step"]').within(() => {
+      cy.get('.MuiStepLabel-iconContainer').should('have.class', 'Mui-active');
+    });
     cy.get('[data-cy="next-button"]').click();
+
+    // Download view
     cy.contains('Download');
+    cy.get('[data-cy="Download-step"]').within(() => {
+      cy.get('.MuiStepLabel-iconContainer').should('have.class', 'Mui-active');
+    });
+    cy.get('[data-cy="complete-annotations-alert"]').should('be.visible');
+    cy.get('[data-cy="datadictionary-preview"]')
+      .should('be.visible')
+      .should('contain', 'Description:"some cool new description"');
+    cy.get('[data-cy="datadictionary-toggle-preview-button"]').click();
+    cy.get('[data-cy="datadictionary-preview"]').should('not.be.visible');
+    cy.get('[data-cy="download-datadictionary-button"]').click();
+
+    const outputFileName = `${exampleDataDictionaryFileName.split('.')[0]}_annotated.json`;
+    cy.readFile(`cypress/downloads/${outputFileName}`).then((fileContent) => {
+      const fileContentString = JSON.stringify(fileContent);
+      expect(fileContentString).to.not.contain('Age of the participant');
```

Comment:
This is because you didn't annotate the "Age" column, right? I would add that as a comment to remind the reader

Reply:
More like because we edited that description so you could say update the annotation.

Reply:
Oh, I see. In that case I'd assert the new description instead. Otherwise this is confusing and I need to keep a lot of info in memory here to understand why you do this negative assert.